packages/jest-each/src/bind.js

```diff
@@ -52,6 +53,11 @@ export default (cb: Function) => (...args: any) => (
   );
 };
 
+const arrayFormat = (str, ...args) => {
+  const matches = (str.match(SUPPORTED_PLACEHOLDERS) || []).length;
+  return util.format(str, ...args.slice(0, matches));
+};
```

Comment:
what about `matches.length > args.lenght`?

Reply:
`.slice` is greedy and will take all of the args still, there will just be some placeholders that aren't replaced. Do we care about this case? I think it should be pretty obvious that the placeholders aren't replaced as there isn't a value for them 😆

Reply:
Should we throw? Probably not

Reply:
Maybe we can add a lint rule for it?

Reply:
I like it 😄